jade/index.jade

```diff
@@ -48,10 +48,10 @@ body
           | This site is fully copied from <a href="http://youmightnotneedjquery.com">youmightnotneedjquery.com</a>, an excellent resource for vanilla JavaSript created by <a href="http://twitter.com/adamfschwartz">@adamfschwartz</a> and <a href="http://twitter.com/zachbloom">@zackbloom</a>.
           | But this time, we take a look at the power of modern native HTML and CSS as well as some of the syntactic sugar of Sass. Because, you might not need scripts for that task at all!
 
-    //- .share-buttons
+    .share-buttons
 
-    //-   .share-button.twitter-share-button
-    //-     iframe(src='http://hubstar.io/twitter/spin?via=HubSpotDev&text=You%20might%20not%20need%20JavaScript&url=http://YouMightNotNeedJS.com&name=%22You%20might%20not%20need%20JavaScript%22', frameborder=0, scrolling=0)
+      .share-button.twitter-share-button
+        iframe(src='http://hubstar.io/twitter/spin?via=HubSpotDev&text=You%20might%20not%20need%20JavaScript&url=http://YouMightNotNeedJS.com&name=%22You%20might%20not%20need%20JavaScript%22', frameborder=0, scrolling=0)
```

Comment:
Hey! Thanks for the PR.

It seems like you just uncommented this code, and the reason i decided to comment it out was because I was trying to not use the http://hubstar.io/twitter/spin app (same with the next issue)